addons/auth_totp_mail_enforce/models/auth_totp_rate_limit_log.py

```diff
@@ -0,0 +1,14 @@
+from odoo import fields, models
+
+
+class AuthTotpRateLimitLog(models.TransientModel):
+    _name = 'auth.totp.rate.limit.log'
+    _description = 'TOTP rate limit logs'
+
+    user_id = fields.Many2one('res.users', required=True, readonly=True)
```

Comment:
Done

Reply:
:+1: `auth_totp_rate_limit_log_user_id_limit_type_create_date_idx` is 59 chars long, that's close to the 64 limit ^^
Could perhaps use `tools.create_index()` to avoid raw SQL, but that's not a big deal.

Reply:
> 👍 `auth_totp_rate_limit_log_user_id_limit_type_create_date_idx` is 59 chars long, that's close to the 64 limit ^^
I know, I tested it just for that case :p. And it was therefore fine